types/network/Network.d.ts

```diff
@@ -971,6 +971,7 @@ export interface EdgeOptions {
   selfReference?: {
     size?: number,
     angle?: number,
+    fullRender?: boolean
```

Comment:
I don't like the `fullRender` name. It's really vague and doesn't give a good idea about the feature it controls. The only thing that comes to my mind right now is `renderBehindTheNode`. Although it is more functional it's not very elegant. Feel free to suggest something else.